test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/GroupMemberGrsControllerSpec.scala

```diff
@@ -80,6 +90,31 @@ class GroupMemberGrsControllerSpec extends ControllerSpec {
       }
     }
 
+    "not store business enity information " when {
+      "registering UK Limited Company with same company number" in {
```

Comment:
Perhaps just say `attempting to register a duplicate organisation` and then it's still valid if we decide to perform the dupe check on other details?

Reply:
I wanted to highlight the use of company number as a criteria which is used to check if it is duplicate.